types/three/src/geometries/ShapeGeometry.d.ts

```diff
@@ -1,13 +1,53 @@
 import { Shape } from './../extras/core/Shape';
 import { BufferGeometry } from './../core/BufferGeometry';
 
+/**
+ * Creates an one-sided polygonal geometry from one or more path shapes.
+ * @example
+ * ```typescript
+ * const x = 0, y = 0;
+ * const heartShape = new THREE.Shape();
+ * heartShape.moveTo(x + 5, y + 5);
+ * heartShape.bezierCurveTo(x + 5, y + 5, x + 4, y, x, y);
+ * heartShape.bezierCurveTo(x - 6, y, x - 6, y + 7, x - 6, y + 7);
+ * heartShape.bezierCurveTo(x - 6, y + 11, x - 3, y + 15.4, x + 5, y + 19);
+ * heartShape.bezierCurveTo(x + 12, y + 15.4, x + 16, y + 11, x + 16, y + 7);
+ * heartShape.bezierCurveTo(x + 16, y + 7, x + 16, y, x + 10, y);
+ * heartShape.bezierCurveTo(x + 7, y, x + 5, y + 5, x + 5, y + 5);
+ * const geometry = new THREE.ShapeGeometry(heartShape);
+ * const material = new THREE.MeshBasicMaterial({
+ *     color: 0x00ff00
+ * });
+ * const mesh = new THREE.Mesh(geometry, material);
+ * scene.add(mesh);
+ * ```
+ * 	* @see {@link https://threejs.org/docs/index.html#api/en/geometries/ShapeGeometry | Official Documentation}
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/src/geometries/ShapeGeometry.d.ts;
  Line: 24;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.